src/tersets.zig

```diff
@@ -52,6 +59,9 @@ pub fn compress(
         .PoorMansCompressionMean => {
             try pmc.compress_mean(uncompressed_values, compressed_values, error_bound);
         },
+        .SwingFilter => {
+            try swing_slide_filter.compress_swing(uncompressed_values, compressed_values, error_bound);
```

Comment:
Maybe add a newline at a proper location as the line is longer than 100 characters.

Reply:
I think it is not longer than 100 characters. I count from the start at `try` to `);` and the line has 91 characters.

Reply:
When talking about the number of characters all white space should be included.

Reply:
Fixed, now the parameters are taking a separate line each.